src/Poseidon/CLI/OptparseApplicativeParsers.hs

```diff
@@ -417,7 +417,7 @@ parseRemoteURL :: OP.Parser String
 parseRemoteURL = OP.strOption (
     OP.long "remoteURL" <>
     OP.help "URL of the remote Poseidon server" <>
-    OP.value "https://c107-224.cloud.gwdg.de" <>
+    OP.value "https://server.poseidon-adna.org" <>
```

Comment:
Yes we could. I could setup any subdomain, like `www.poseidon-adna.org`... but we need to think of a clever transition period or redirect to not alienate users who grew fond of the GitHub.io URL.

Reply:
Good point. Here's the relevant GitHub documentation for reference: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site

Reply:
I will open an issue for this. I don't actually know whether the old URL will then still work but automatically redirect. That would be good. If it stops working it would be bad.

Reply:
https://github.com/poseidon-framework/poseidon-framework.github.io/issues/42